work_cycles.md

```diff
@@ -0,0 +1,68 @@
+# DLS Work Cycles
+
+## Development Rhythm
+
+Our team works in a 3 week rhythm. The first week is Maintenance & Research and
+the following two weeks are work cycle weeks.
+
+The following table represents our cycle-related meeting schedule. Each meeting
+will be described below.
+
+|                        | Monday         | Tuesday | Wednesday | Thursday | Friday  |
+| ---------------------- | -------------- | ------- | --------- | -------- | ------- |
+| Research & Maintenance | Cycle Planning | Standup | Standup   | Standup  | Standup |
+| Cycle Work (Week 2)    | Standup        | Standup | Standup   | Standup  | Standup |
+| Cycle Work (Week 3)    | Standup        | Standup | Standup   | Standup  | Wrap-up |
+
+## Meetings
+
+### Cycle Planning
+
+Occurs Monday at 3:00 PM eastern of Research & Maintenance week. Participants are
+members of DLS and the product owner for the upcoming project.
+
+**Work Cycle Board**:
+[Zenhub](https://app.zenhub.com/workspaces/dls-work-cycle-613924a1df719e0013b678b0/board?repos=98223070)
+
+#### Goals
+
+1. Determine the tickets which will be undertaken in the next two weeks. Give
+   them the 'work-cycle' label.
+1. If this is a feature sprint, PO will identify one ticket which is a
+   meaningful enhancement to an existing feature for users. Give it the
+   work-cycle label.
+1. Determine if there's any research to be done to validate that the work can
+   happen in the following two weeks.
+
+
+### Standup
+
+Occurs daily at 3:30 PM Eastern when there isn't also a Cycle Planning or
+Wrap-up meeting. Participants are members of DLS.
+
+**Research & Maintenance Board**:
```

Comment:
It's not clear to me why this is between standup description and goals?

Reply:
Sorry, I didn't catch that before I merged it. Unless anyone objects, I'm going to move it down below the last bullet in the Wrap Up meeting since that seems most relevant.

Reply:
👍🏻 thanks @sdellis !